docs/backups-ondemand.md

```diff
@@ -1,59 +1,121 @@
-    spec:
-      clusterName: my-cluster-name
-      storageName: s3-us-west
-      type: logical
-    ```
+To create a Backup resource, you need a special custom resource manifest. The [deploy/backup/backup.yaml :octicons-link-external-16:](https://github.com/percona/percona-server-mongodb-operator/blob/main/deploy/backup/backup.yaml) is the example manifest that you can use.
+{.power-number}
+
+1. Specify the following configuration:
+
+    * `metadata.name` is the name of the backup. You will need this name when you [restore from this backup](backups-restore.md). The default name is `backup1`.
+
+    * `spec.clusterName` is the name of your cluster (prior to
+        the Operator version 1.12.0 this key was named `spec.psmdbCluster`). Run `kubectl get psmdb -n <namespace>` to find out the cluster name.
+
+    * `spec.storageName` is the name of your [already configured storage](backups-storage.md).
+
+    * `spec.type` is the [backup type](backups.md#backup-types). If you leave it empty, the Operator makes a logical backup by default.
+
+    **Examples**
+
+    === "Logical"
+   
+        ```yaml
+        apiVersion: psmdb.percona.com/v1
+        kind: PerconaServerMongoDBBackup
+        metadata:
+          finalizers:
+          - percona.com/delete-backup
+          name: backup1
+        spec:
+          clusterName: my-cluster-name
+          storageName: s3-us-west
+          type: logical
+        ```
+
+    === "Physical"
+    
+        ```yaml
+        apiVersion: psmdb.percona.com/v1
+        kind: PerconaServerMongoDBBackup
+        metadata:
+          finalizers:
+          - percona.com/delete-backup
+          name: backup1
+        spec:
+          clusterName: my-cluster-name
+          storageName: s3-us-west
+          type: physical
+        ```
+
+    === "Incremental"
+
+        To make incremental backups, consider the following:
+
+        1. Make the incremental base backup first. The Operator needs the base to start the chain of increments and save only changes from previous backup. 
+
+        2. Use the same storage for base backup and increments. 
+
+        3. The `percona.com/delete-backup` finalizer and the [` .spec.backup.tasks.[].keep`](operator.md##backuptaskskeep) option are not available for this backup type.
+
+        Here's the configuration example for the base incremental backup
+    
+        ```yaml
+        apiVersion: psmdb.percona.com/v1
+        kind: PerconaServerMongoDBBackup
+        metadata:
+          finalizers:
+          - percona.com/delete-backup
+          name: backup1
+        spec:
+          clusterName: my-cluster-name
+          storageName: s3-us-west
+          type: incremental_base
+        ```
 
-3. Run the actual backup command using this file:
+        This configuration example is for subsequent incremental backups:
+
+        ```yaml
+        apiVersion: psmdb.percona.com/v1
+        kind: PerconaServerMongoDBBackup
+        metadata:
+          finalizers:
```

Comment:
Please remove finalizers from example (as it was mentioned - we ignore delete-backup finalizer for incremental so it is better not to have it here IMHO)